tests/fast-integration/test/k8s-test.js

```diff
@@ -0,0 +1,55 @@
+const k8s = require("@kubernetes/client-node");
+const { deploymentYamlString } = require("./fixtures");
+const { expect } = require("chai");
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+
+const k8sDynamicApi = kc.makeApiClient(k8s.KubernetesObjectApi);
+const deployObj = k8s.loadYaml(deploymentYamlString);
+
+describe("dummy test", function () {
+  this.timeout(10000);
+
+  before(async function () {
+    try {
+      await k8sDynamicApi.delete(deployObj);
+    } catch (error) {
+      console.warn(error.body.message);
+    }
+  });
+
+  it("create deployment", async function () {
```

Comment:
I think it should go to `before` section. The idea is that deployments, functions are created before tests not during the test. In the test we should check if deployment has status ready.